### PR TITLE
Added __str__ method and admin.py

### PR DESCRIPTION
**Pull request by travistruett**
@Wilduck @wesleykendall can you take a look at this?

### CODE REVIEW

File: localized_recurrence/models.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from dateutil.relativedelta import relativedelta
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
```
Comment (travistruett) [inline localized_recurrence/models.py]: https://docs.djangoproject.com/en/1.6/ref/utils/#module-django.utils.encoding

@micahhausler not sure if you've been running into this when upgrading everything to Python3, but what has been your preferred method of handling both __str__ and __unicode__ model methods? This seems to be pretty standard, at least it's what Django itself uses.

File: setup.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def get_version():
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     ],
     install_requires=[
-        'Django>=1.6',
+        'Django>=1.6,<1.8',
```
Comment (travistruett) [inline setup.py]: Until Django Nose is fixed...

**Comment by travistruett** [comment]
It's failing because code coverage is at 99%.

Do I need to test the admin.py or should that be excluded somehow? I only ask because I don't have any custom methods, it's just vanilla code.

File: setup.py (hunk 2 above)
Comment (micahhausler) [inline setup.py]: Could you leave this how it was? It actually makes it more complicated for when we upgrade. TravisCI has the appropriate versions pinned for testing https://github.com/ambitioninc/django-localized-recurrence/blob/develop/.travis.yml#L10-L11

File: localized_recurrence/models.py (hunk 1 above)
Comment (micahhausler) [inline localized_recurrence/models.py]: This looks good. I typically import that decorator directly from [`six`](http://pythonhosted.org//six/#six.python_2_unicode_compatible)

**Comment by micahhausler** [comment]
@travistruett just add `localized_recurrence/admin.py` to the [`.coveragerc` omit line](https://github.com/ambitioninc/django-localized-recurrence/blob/develop/.coveragerc#L7)

File: setup.py (hunk 2 above)
Comment (travistruett) [inline setup.py]: Gotcha, yea I can remove it... but it's technically broken for anybody that's wanting to test locally which is kind of a pain.

**Comment by travistruett** [comment]
@wesleykendall should be good to go now. Ignore the despair listed above.

**Comment by wesleykendall** [comment]
@travistruett looks good, I'll upgrade the version and release it

**Pull request merged**